controllers/articles.controller.js

```diff
@@ -0,0 +1,16 @@
+const { selectArticleById } = require('../models/articles.model');
+
+
+exports.getArticleById = (req, res, next) => {
+  const article_id = Number(req.params.article_id)
+  
+  if (isNaN(article_id)) {
+    return next({ status: 400, msg: 'Invalid data type'})
+  }
```

Comment:
This doesn't need to be here. If the article_id is not a number, SQL will throw an error, so you can use the error code in your PSQL error handler